src/app/page.js

```diff
@@ -38,14 +39,20 @@ const Home = () => {
         </div>
 
         <div className="flex sm:flex-row-reverse flex-wrap items-center justify-center my-[2rem]">
-          <div className="sm:flex-1 text-center sm:text-right mb-10 sm:my-0">
-            <h1 className="text-lg md:text-xl">a team of</h1>
-            <h1 className="text-lg md:text-xl">passionate</h1>
-            <h1 className="font-semibold text-[2.5rem] md:text-[3.5rem] lg:text-[4rem]">
+          <div className="sm:flex-1 text-center sm:text-right mb-10 sm:my-0 leading-[3.75rem] sm:leading-[4rem] md:leading-[5rem]">
+            <p className="font-semibold text-[2rem] md:text-[2.5rem] lg:text-[3rem] text-DevDogBlue leading-3">
+              a team of
+            </p>
+            <p className="font-semibold text-[2rem] md:text-[2.5rem] lg:text-[3rem]  text-DevDogBlue">
+              passionate
+            </p>
+            <h1 className="font-extrabold text-[3rem] md:text-[4rem] lg:text-[5rem] text-MidnightBlue">
```

Comment:
I think "Student" does not need to be bolded, just in a larger font size, because the Role titles in the following line does not seems to be bolded.
![image](https://github.com/user-attachments/assets/b3eedd03-dd96-4de3-98e7-f8f5ac68ce6b)

Reply:
I just made them both the same font, the student text is bolder than the roles. I made them both font-semibold